src/app/Services/data-json.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class JsonDataService {
```

Comment:
You have a user service doing the same job as this service, do you have any future plans for this service?